kits/japanese-teacher/prompts/quiz_instructor-llmnode-355_user_1.md

```
@@ -0,0 +1,5 @@
+MODE: GENERATE_QUIZLEVEL:{{triggerNode_1.output.context}}
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)